uninstall.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+#
+# uninstall/removes g910-gkeys.
+#
+# options:
+#  -a: will also remove /etc/g910-gkeys/ configuration directory.
+#  -d: dry-run: will display actions without doing anything
+#
+# TODO: remove possible faulty previous versions files (in /etc/systemd/system
+#       and /usr/lib/systemd/system)
+#
+# BUGS: files.txt is not correct:
```

Comment:
Files is automatically generated. For better way you would probably need to write files installed manually.

Reply:
?? I added something to remove faulty hard-coded early g910-gkeys systemd (/etc/systemd and /usr/lib/systemd).